skycoin/skywire
# Move to united binary

Did you run `make format && make check`? Yes

Fixes #1769 

 Changes:	
- use merged binary in windows and mac
- update goreleaser and installers scripts

How to test this PR:
- use `make build-merged` or `build-merged-windows` based on you os
- in `/build` folder, trying to generate config by `skywire cli config gen` or `skywire.exe cli config gen`
- check configs in launcher apps part
- run visor by `skywire visor -c skywire-config.json` or with `.exe` extension on Windows
- everything should work as well as
- copy old config there, regenerate by `-r` flag and check apps part again

Note: release and installer will test after merging this PR
